### PR TITLE
Re-define MP_RTT Age parameter

### PR DESCRIPTION
**Pull request by markusa**

### CODE REVIEW

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 1]
```diff
@@ -709,7 +709,7 @@ the token and nonce of the MP_JOIN for which authentication shall be performed.
 
 The MP_RTT option is used to transmit RTT values in milliseconds and
 MUST belong to the path over which this information is transmitted.
-Additionally, the age of the measurement is specified in milliseconds.
+Additionally, the age of the last measurement is specified in milliseconds.
 
 The RTT and Age information is a 32-bit integer, which allows to cover a period of
```
Comment (shjohnson-bt) [inline draft-ietf-tsvwg-multipath-dccp.mkd:714]: Reword:

"The RTT and Age information are 32-bit integers, providing a range of approximately1193 hours"

**Review by shjohnson-bt — CHANGES_REQUESTED** [review]

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 2]
```diff
@@ -709,7 +709,7 @@ the token and nonce of the MP_JOIN for which authentication shall be performed.
 
 The MP_RTT option is used to transmit RTT values in milliseconds and
 MUST belong to the path over which this information is transmitted.
-Additionally, the age of the measurement is specified in milliseconds.
+Additionally, the age of the last measurement is specified in milliseconds.
```
[the remark this answers is not in the document]
Comment (markusa) [reply draft-ietf-tsvwg-multipath-dccp.mkd:712]: I used "last" intentionally to specify the time span from previous (last) conducted RTT measurement till "now" - creation of MP_RTT option. The idea is to let receiver justify if a measurement is up to date enough to be used in the re-ordering process or is outdated.

Please suggest some better wording if you feel the text need improvement.

[the remark this answers is not in the document]
Comment (shjohnson-bt) [reply draft-ietf-tsvwg-multipath-dccp.mkd:712]: Reading on, the definition of age is given below and so I don't think we need give any detail on what it means here other than its units

**Review by maramend — CHANGES_REQUESTED** [review]

File: draft-ietf-tsvwg-multipath-dccp.mkd (hunk 2 above)
[the remark this answers is not in the document]
Comment (markusa) [reply draft-ietf-tsvwg-multipath-dccp.mkd]: see https://github.com/markusa/ietf-multipath-dccp/pull/81/commits/5ee9a808cfba4bb9d0c935019c2bfab3184169f8

File: draft-ietf-tsvwg-multipath-dccp.mkd (hunk 1 above)
Comment (markusa) [reply draft-ietf-tsvwg-multipath-dccp.mkd]: see https://github.com/markusa/ietf-multipath-dccp/pull/81/commits/5ee9a808cfba4bb9d0c935019c2bfab3184169f8

**Review by shjohnson-bt — APPROVED** [review]

**Review by NathalieRM — APPROVED** [review]

**Pull request merged**